README.md

```diff
@@ -69,7 +69,7 @@ client = WsiDicomWebClient.create_client(
 slide = WsiDicom.open_web(
     client,
     "study uid to open",
-    "series uid top open"
+    "series uids top open"
```

Comment:
This would be either `"series uid op open"` or `["series uid 1 to open", "series uid 2 top open"]` right?